css/index.css

```diff
@@ -1,3 +1,390 @@
+	display: flex;
+	font-size: 6rem;
+	padding-top: 1em;
+
+}
+
+.cta button {
+
+
+	 background-color: white;
+	 color: black;
+	 width: 60%;
+	 height: 2rem;
+
+}
+
+.middle-sec {
+
+	display:flex;
+	padding-top: 1em;
+	justify-content: space-between;
+
+
+
+}
+.middle-sec .features {
+
+	display: flex;
+	flex-direction: column;
+  padding-left: 1em;
+	width: 35%;
+
+
+}
+.middle-sec .about {
+
+display: flex;
+flex-direction: column;
+padding-right: 1em;
+width: 35%;
+
+}
+.bottom-img {
+
+	 display:flex;
+	 justify-content: space-around;
+	 width: 100%;
+
+}
+.bottom-img img {
+
+width: 100%;
+}
+
+.bottom-sec {
+	display: flex;
+	justify-content: space-between;
+	padding-top: 1em;
+
+}
+.bottom-sec .services {
+
+	display: flex;
+	width: 25%;
+	flex-direction: column;
+	padding-left: 1em;
+
+}
+
+.bottom-sec .product {
+
+display: flex;
+width: 25%;
+flex-direction: column;
+
+}
+
+.bottom-sec .vision  {
+
+	display: flex;
+	width: 25%;
+	flex-direction: column;
+	padding-right: 1em;
+
+
+
+
+}
+
+.contact h4 {
+	padding-top: 10px;
+
+}
+
+
+
+footer {
+
+	text-align: center;
+
+}
+
+
+
+
+@media (max-width: 1024px) {
+
+	body {
+		max-width: 1024px;
+		margin: 20px;
+	
+	}
+	header hr {
+		display: none;
+	}
+
+.cta {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: column;
+
+
+}
+.cta button {
+	padding-left: 20px;
+	margin-left: 64px;
+		margin-top: 20px;
+		width:46%;
+}
+.cta h1 {
+	font-size: 70px;
+}
+.Contact {
+	padding-top: 10px;
+	padding-left: 10px;
+}
+}
+	
+
+@media (max-width: 800px) {
+
+	body {
+		max-width: 800px;
+		max-height: 1370px;
+		margin: 20px;
+		resize: both;
+	
+	}
+	header{
+	display: flex;
+	flex-wrap:nowrap;
+	flex-direction: column-reverse;
+	}
+	header hr {
+		display: none;
+	}
+	nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding-top: 20px;
+    text-decoration: none;
+
+	}
+	nav a {
+		color: lightgrey;
+	}
+	header img {
+
+		display:flex;
+		padding-left:185px;
+	}
+	.top-sec img {
+		display: none;
+	}
+	.cta button {
+		width: 57%;
+
+	}
+	.bottom-sec {
+		display: flex;
+		justify-content: space-between;
+		padding-top: 1em;
+		height: max-content;
+		width: auto;
+	}
+	.bottom-img img {
+		width: 102%;
+		height: 50%;
+	}
+
+	
+	hr {
+		background-color:black;
+		color: black;
+		border: solid 3px black;
+		height: 3px;
+	}
+	.Contact{
+		text-align: center;
+		
+	}
+	.Contact a {
+		text-decoration: none;
+		color:red;
+		background: black;
+	}
+
+
+}
+@media (max-width: 500px) {
+    
+	body {
+		max-width: 500px;
+		max-height: 2047px;
+		resize: both;
+	
+	}
+	header{
+		display: flex;
+		flex-wrap:nowrap;
+		flex-direction: column-reverse;
+		}
+	
+header img {
+	align-self: center;
+	padding-left: 20px;
+}
+.navigation-main {
+
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+}
+  a {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	align-self: stretch;
+	align-items: center;
+	padding: 20px;
+	padding: left 50px;
```

Comment:
```suggestion

```

Reply:
Should start to use less of padding/margin for major positioning; fine for details and minor adjustments